### ISSUE TITLE
Bump traefik to v3

### ISSUE THREAD

**Issue by joecorall** [Islandora-Devops/isle-site-template#51]
Should follow this guide for regressions then do a major bump is all is well:
https://doc.traefik.io/traefik/v3.0/migration/v2-to-v3/

**Comment by joshdentremont** [comment]
I just went through the guide and I don't see anything that needs changing in the Site Template docker-compose.yml. I am able to spin up a new site using traefik 3.1.0 and it all looks OK to me. I am having issues connecting to Solr on the newly spun up site, but that seems to be unrelated because it is happening with traefik 2.10.5 as well. 

On my existing site, I updated the docker-compose.yml to 3.1.0 and the site seems to function as normal. I would say that all we need to do is update the traefik image and we are good to go. I will test it on my staging site tomorrow, and put in a PR to update it so that it can be tested by others.

<img width="1485" alt="image" src="https://github.com/user-attachments/assets/15fc665e-c649-4fbc-8c96-3b515ae0dd05">
